cdc-accelerator/feature/acl.js

```diff
@@ -39,18 +39,44 @@ export default class ACL extends Feature {
         })
     }
     reset() {
-        //This is supposed to be empty
+        // The method is empty because there is nothing left to do after PermissionGroups.reset is executed
     }
-    async deploy(partnerDirectory, siteInfo) {
-        console.log('deploy was called')
+    async deploy(permissionGroupDirectory, siteInfo) {
+        const buildDirectory = path.join(permissionGroupDirectory, this.getName())
+        const aclFiles = fs.readdirSync(buildDirectory)
+        for (let aclFile of aclFiles) {
+            const aclData = fs.readFileSync(path.join(buildDirectory, aclFile), { encoding: 'utf8' })
+
+            if (!aclData || !aclData.length) {
+                throw new Error(`Invalid file: ${aclFile}`)
+            }
+            const parsedData = JSON.parse(aclData)
+            const aclName = path.parse(aclFile).name
+            let response = await this.deployAclRequest(siteInfo.dataCenter, aclName, siteInfo.partnerId, parsedData, this.#credentials)
+            if (response.errorCode) {
+                throw new Error(JSON.stringify(response))
+            }
+        }
     }
 
+    async deployAclRequest(dataCenter, aclID, partnerID, aclContent, credentials) {
```

Comment:
the method deployAclRequest can be removed, it is not doing anything than calling other method. Or copy the content of setAclRequest to this method